### PR TITLE
feat(docs): add accessibility section for `pagination` component

### LINKED ISSUE #1569: Add accessibility section for `pagination`
### Subject

accessibility section for `pagination`

#### Create an accessibility section referring to the links below
https://yamada-ui.com/components/disclosure/tabs/accessibility#aria-roles-and-attributes ( https://github.com/yamada-ui/yamada-ui/issues/1552 )
https://mui.com/material-ui/react-pagination/#accessibility

#### Include the following content
* aria, role , tabindex property are supported

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by mei28**
Closes #1569 

## Description

[HELP] 
I'm sorry, I'm not sure how to adapt the changed content to the document. As a result, I haven't been able to provide a screenshot, and I haven't confirmed how the changes are displayed myself.

Also, I'm not certain if I’ve included everything I needed in the document. Please let me know if there’s anything missing or if there’s too much.

### CODE REVIEW

File: docs/contents/components/navigation/pagination/accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Pagination
+description: "`Pagination` is a component for managing the pagination and navigation of content."
+package_name: "@yamada-ui/pagination"
+is_tabs: true
```
Comment (108yen) [inline docs/contents/components/navigation/pagination/accessibility.mdx:5]: Please remove `is_tabs`.

**Review by 108yen — CHANGES_REQUESTED** [review]
I think table content is something wrong.
refarence: https://yamada-ui.com/ja/components/data-display/stat/accessibility

**Comment by mei28** [comment]
English version:
image

Japanese version:
image

File: docs/contents/components/navigation/pagination/accessibility.mdx (hunk 1 above)
Comment (mei28) [reply docs/contents/components/navigation/pagination/accessibility.mdx]: Thank for your follow. I had removed it.

**Comment by 108yen** [comment]
@mei28 
Check this out too.

> I think table content is something wrong.
refarence: https://yamada-ui.com/ja/components/data-display/stat/accessibility

**Comment by mei28** [comment]
Do I only need to document the section below in this case?
https://github.com/yamada-ui/yamada-ui/blob/74c0ae30a7cd171e01dd52582c8108dbe45c6a3b/packages/components/pagination/src/pagination.tsx#L128-L130

**Comment by 108yen** [comment]
@mei28 
I think format of the table contents is wrong.

**Review by taiseiiiii — DISMISSED** [review]
@mei28 
I guess this document should have a section "Keyboard Navigation"
I discover the description related to this in [this page](https://mui.com/material-ui/react-pagination/#accessibility)

> Keyboard
> The pagination items are in tab order, with a tabindex of "0".

**Comment by mei28** [comment]
@hirotomoyamada 
Do I need to include the content from the link regarding keyboard accessibility, like in the following example: https://mui.com/material-ui/react-pagination/#accessibility?

**Comment by hirotomoyamada** [comment]
@mei28 

Hi, either is fine. It is cooler if it is stated.

**Review by hirotomoyamada — CHANGES_REQUESTED** [review]
Please add `"Go to first page"`, `"Go to previous page"`, `"Go to next page"` and `"Go to last page"` of `aria-label` to the explanation.

File: docs/contents/components/navigation/pagination/accessibility.ja.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+tab: アクセシビリティ
+order: 3
+---
+
+## ARIAロールと属性
+
+| コンポーネント | ロールと属性        | 使い方                                                                                                                                        |
+| -------------- | ------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Pagination`   | `role="navigation"` | ページ間のナビゲーション領域であることを示します。                                                                                            |
+|                | `aria-label`        | `page`に`"ellipsis"`が設定されている場合は`"Jump to omitted pages"`を設定し、`page`が設定されている場合は`"Go to page ${page}"`を設定します。 |
```
Comment (hirotomoyamada) [inline docs/contents/components/navigation/pagination/accessibility.ja.mdx:15]: The `aria-label` is set to the `PaginationItem` internal component.

Comment (mei28) [reply docs/contents/components/navigation/pagination/accessibility.ja.mdx:15]: fixed from `Pagination`  into `PaginationItem` and added `Internal` tag

File: docs/contents/components/navigation/pagination/accessibility.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+tab: Accessibility
+order: 3
+---
+
+## ARIA Roles and Attributes
+
+| Component    | Role and Attributes | Usage                                                                                                                            |
+| ------------ | ------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `Pagination` | `role="navigation"` | Indicates that this is a navigation region between pages.                                                                        |
+|              | `aria-label`        | If `page` is set to `"ellipsis"`, `"Jump to omitted pages"` is set, and if `page` is set to page, `"Go to page ${page}"` is set. |
```
[the remark this answers is not in the document]
Comment (mei28) [reply docs/contents/components/navigation/pagination/accessibility.mdx:15]: fixed from `Pagination`  into `PaginationItem` and added `Internal` tag

**Comment by mei28** [comment]
I added a style for `em` inside `code` as follows:

```typescript
<Text
  as="code"
  apply="mdx.code"
  dangerouslySetInnerHTML={{ __html: props.children }}
/>
```

I'm not sure if this is the correct approach, so if there’s a better way, please let me know.

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**